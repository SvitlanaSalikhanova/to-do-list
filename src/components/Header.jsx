import React from 'react';
import HeaderCss from '../styles/Header.module.scss';

export default function Header() {
    return (
        <div className={HeaderCss.header} />
    );
}
