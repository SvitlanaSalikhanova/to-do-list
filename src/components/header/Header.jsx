import React from 'react';
import HeaderCss from './Header.module.scss';

export default function Header() {
    return (
        <div className={HeaderCss.header} />
    );
}
