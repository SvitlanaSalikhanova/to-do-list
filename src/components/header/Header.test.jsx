import React from 'react';
import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import Header from './Header';

describe('DeleteButton component tests', () => {
    it('Matches Snapshot', () => {
        const domTree = renderer.create(
            <Header />,
        ).toJSON();
        expect(domTree).toMatchSnapshot();
    });

    it('header renders', () => {
        const { container } = render(
            <Header />,
        );
        const headers = container.getElementsByClassName('header');
        expect(headers.length).toBe(1);
    });
});
