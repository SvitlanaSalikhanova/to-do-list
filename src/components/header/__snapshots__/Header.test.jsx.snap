// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeleteButton component tests Matches Snapshot 1`] = `
<div
  className="header"
/>
`;
